Sources/ClientRuntime/Networking/Http/CRT/CRTClientEngine.swift

```diff
@@ -129,7 +129,7 @@ public class CRTClientEngine: HttpClientEngine {
         switch connection.httpVersion {
         case .version_1_1:
             self.logger.debug("Using HTTP/1.1 connection")
-            let crtRequest = try request.toHttpRequest()
+            let crtRequest = try request.toHttpRequest(escaping: false)
```

Comment:
do we also need to do this for the http2 requests which we create in two places below?

Reply:
Yes we do, good callout.  I'm going to change the `toHTTPRequest()` method to default to not escaping, though, since escaping is more the "special case" than not escaping.